packages/wdio-sauce-service/src/launcher.js

```diff
@@ -1,19 +1,19 @@
-import { promisify } from 'util'
 import { performance, PerformanceObserver } from 'perf_hooks'
 
 import logger from '@wdio/logger'
-import SauceConnectLauncher from 'sauce-connect-launcher-update'
+import SauceLabs from 'saucelabs'
 
-function modifyObject (root, propertiesObject) {
-    for (const [k, v] of Object.entries(propertiesObject)) {
-        root[k] = v
-    }
-}
+const SC_RELAY_DEPCRECATION_WARNING = [
```

Comment:
Add newlines at the end?

Reply:
this will be logged into the log file or stdout and doesn't need a new line.